docs/general/networking/nginx.md

```diff
@@ -423,39 +423,4 @@ LOCATION: /
 
 ```
 
-```config
-
-LOCATION: /socket
-
-    # Proxy Jellyfin Websockets traffic
-    proxy_set_header Connection "upgrade";
-    proxy_set_header X-Real-IP $remote_addr;
-    proxy_set_header X-Forwarded-Protocol $scheme;
-    proxy_set_header X-Forwarded-Host $http_host;
-    
-    # Security / XSS Mitigation Headers
-    # NOTE: X-Frame-Options may cause issues with the webOS app
-    add_header X-Frame-Options "SAMEORIGIN";
-    add_header X-XSS-Protection "0";
-    add_header X-Content-Type-Options "nosniff";
-
-    # Content Security Policy
-    # See: https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
-    # Enforces https content and restricts JS/CSS to origin
-    # External Javascript (such as cast_sender.js for Chromecast) must be whitelisted.
-    # NOTE: The default CSP headers may cause issues with the webOS app
-    #add_header Content-Security-Policy "default-src https: data: blob: http://image.tmdb.org; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' https://www.gstatic.com/cv/js/sender/v1/cast_sender.js https://www.gstatic.com/eureka/clank/95/cast_sender.js https://www.gstatic.com/eureka/clank/96/cast_sender.js https://www.gstatic.com/eureka/clank/97/cast_sender.js https://www.youtube.com blob:; worker-src 'self' blob:; connect-src 'self'; object-src 'none'; frame-ancestors 'self'";
-
-```
```

Comment:
Is this not needed anymore? Normally without additional configurations Nginx does not work with WebSockets.

Reply:
> The playback tracker in the dashboard updated as expected.

That does not use websockets though. The best way to test sockets is probably with the remote control feature, so try to play/pause/seek a different device. And make sure that it still works after 5 minutes to verify there is no timeout in Nginx.

Reply:
Confirmed that enabling "Websocket Support" on the details page is sufficient to open and maintain websockets.  I was able to remote control another session as expected.